src/components/ui/FishTable.jsx

```diff
@@ -30,6 +31,17 @@ class FishTable extends React.Component {
 
   filterWithName = (selected, test) => test.toLowerCase().includes(selected.toLowerCase());
 
+  get profMulti() {
+    const { fisher, angler } = this.props;
+
+    if (fisher) {
+      return 1.25;
+    } else if (angler) {
+      return 1.5;
+    }
+    return 1;
```

Comment:
Why not

```
if (fisher) {
  return 1.25;
}

if (angler) {
  return 1.50;
}

return 1;
```

Reply:
Lack of a functional difference. both work for this case.